script.js

```diff
@@ -1,4 +1,115 @@
+// ========== Reversed Scissors Paper Stone ========== //
+
+// The user inputs one of "reversed scissors", "reversed paper", or "reversed stone".
+// The program randomly choose "reversed scissors", "reversed paper", or "reversed stone" by rolling a dice.
+
+var rollDigit = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.ceil(randomDecimal);
+  return randomInteger;
+};
+
+// If number is 1, it means reversed scissors.
+// If number is 2, it means reversed stone.
+// If number is 3, it means reversed paper.
+
+var assignInteger = function () {
+  var digit = rollDigit();
+  if (digit == 1) {
+    var choice = "Reversed Scissors";
+  }
+  if (digit == 2) {
+    var choice = "Reversed Stone";
+  }
+  if (digit == 3) {
+    var choice = "Reversed Paper";
```

Comment:
Only need to call var choice once to declare that choice is a var. subsequently can just use choice = value to reassign it. you can declare var choice ; without assigning a value once ,above all the if conditions